### DISCUSSION TITLE
globalData的一些疑问

### DISCUSSION THREAD

**Discussion by whyisthat** [kbengine/kbengine#1381] [Q&A]
请问一下：
class player（KBEngine.Entity,GameObject）:
  def _init_(self):
      ...
      KBEngine.globalData["player"] = self
      self.Room ={}

1.注册了全局之后，跨baseApp远程访问player里的方法 是否就不需要在def里定义了？
2.跨baseApp 是否可以直接调用 KBEngine.globalData["player"].Room[]?
3.同2疑问，看到API里说到只有顶层的值才会被广播，也就是说Room值发生改变，远程访问Room值不会改变？

**Comment by worldg** [comment]
当然要定义， 存到globalData后在其他进程上实际存的是个entitycall

**Reply by whyisthat** [reply]
感谢大佬回复
